meeting_minutes/2024/2024-05-29.md

```diff
@@ -22,18 +22,17 @@ Quorum was not reached due to inability to register attendees due to OASIS syste
 | Dina       | Truxius     | Federal Office for Information Security (BSI)               | Voting Member               |
 | Sonny      | van Lingen  | Huawei Technologies Co., Ltd.                               | Voting Member               |
 | Rhonda     | Levy        | Cisco                                                       | Voting Member               |
-| Christoph   | Plutte      | Ericsson                                                    | Member                      |
+| Christoph  | Plutte      | Ericsson                                                    | Member                      |
```

Comment:
Is Christoph Plutte a "permanent non-voter"? Otherwise, I guess he might have retrieved voting rights as he is also listed present in meeting minutes of 2024-01-31 and 2024-02-28. (Don't rely on me here - I did not check which of those counted towards voting eligibility...)